cmd/memcached_exporter/main.go

```diff
@@ -66,7 +67,18 @@ func main() {
 		if *serverName == "" {
 			*serverName, _, err = net.SplitHostPort(*address)
 			if err != nil {
-				level.Error(logger).Log("msg", "Error running HTTP server", "err", err)
+				if strings.HasPrefix(*address, string(os.PathSeparator)) {
```

Comment:
Hrmm, this is suspect. The path to the socket could be relative to the current directory… I just went down a 🐰 🕳️  to figure out how the address is used to decide what kind it is, and while I don't like it, I see that the actual decision [happens here](https://github.com/grobie/gomemcache/blob/239240bbc445c4f056ebbc16829a8f19da834446/memcache/selector.go#L56) and uses `Contains(…, "/")`. For consistency, let's use the same condition here?

```suggestion
				// Using the same condition as the memcached client library: https://github.com/grobie/gomemcache/blob/239240bbc445c4f056ebbc16829a8f19da834446/memcache/selector.go#L56
				if strings.Contains(server, "/") {
```